hack/bicep-types-radius/generated/index.json

```diff
@@ -7,19 +7,19 @@
       "$ref": "applications/applications.core/2023-10-01-preview/types.json#/119"
     },
     "Applications.Core/environments@2023-10-01-preview": {
-      "$ref": "applications/applications.core/2023-10-01-preview/types.json#/154"
+      "$ref": "applications/applications.core/2023-10-01-preview/types.json#/157"
```

Comment:
index.json and types.json were just out of date so updated these

Reply:
Did we miss running make generate or something else that you had to do update these?

Reply:
There were changes to the typespec schema that happened after make generate was originally run. So just reran to get this up to date with the latest changes